.binder/environment.yml

```diff
@@ -2,16 +2,9 @@ name: 95199651-9e81-4cae-a3a7-66398a9a5f62
 channels:
```

Comment:
I suggest we keep all relevant dependencies in `environment.yml`. This might facilitate users uncommenting the cells, so they don't need to worry about missing libraries.

```
name: 95199651-9e81-4cae-a3a7-66398a9a5f62
channels:
  - conda-forge
dependencies:
  - python
  - jupyter
  - pooch
  - py7zr
  - intake
  - intake-xarray
  - scikit-image
  - xmltodict
  - opencv
  - matplotlib
  - pytorch
  - torchvision
  - huggingface_hub
```